modules/EasyAbp.PaymentService.WeChatPay/src/EasyAbp.PaymentService.WeChatPay.Domain/EasyAbp/PaymentService/WeChatPay/PaymentServiceWeChatPayHandler.cs

```diff
@@ -51,7 +54,7 @@ public virtual async Task HandleAsync(XmlDocument xmlDocument)
             payment.SetExternalTradingCode(reader["transaction_id"] ??
                                            throw new XmlDocumentMissingRequiredElementException("transaction_id"));
             
-            if (reader["result_code"] == "SUCCESS")
+            if (context.IsSuccess)
```

Comment:
`context.IsSuccess` is the response for WeChat.

Reply:
I will fix this now.